client/packages/invoices/src/OutboundShipment/DetailView/Toolbar.tsx

```diff
@@ -105,12 +107,24 @@ export const Toolbar: FC<{
             <DropdownMenuItem IconComponent={ZapIcon} onClick={onAllocate}>
               {t('button.allocate-lines')}
             </DropdownMenuItem>
-            <DropdownMenuItem
-              IconComponent={ArrowLeftIcon}
-              onClick={() => onReturnLines(selectedIds)}
+
+            <Tooltip
+              title={
+                !selectedStockLineIds.length
+                  ? t('message.placeholder-lines-cannot-be-returned')
+                  : ''
+              }
             >
-              {t('button.return-lines')}
-            </DropdownMenuItem>
+              <span>
+                <DropdownMenuItem
+                  disabled={!selectedStockLineIds.length}
```

Comment:
Outbound shipment placeholder lines also don't have stock lines, so can't be selected to return - I did put a tooltip here to explain as it didn't seem obvious to me, but curious whether you think it is needed?

Reply:
Probably also shouldn't be able to return from OutboundShipment until after it is shipped? But I think we were procrastinating putting in some of this logic... so I was just trying to handle the cases where you select rows but don't end up with any connected stock lines!

Reply:
I am kind of moving away from thinking tooltips, since we can't really do it for mobile (and that's actually our primary focus, at least that's where the most deployment will be).

I think we would have warning when this is clicked, when status is not 'Shipped' or 'Delivered' or 'Verified', saying can only return outbound that was shipped, cannot have placeholder lines in those statuses, so we wouldn't need this warning